Yandex/YandexAdapter/ALYandexMediationAdapter.m

```diff
@@ -89,6 +92,10 @@ - (void)initializeWithParameters:(id<MAAdapterInitializationParameters>)paramete
     
     [self updateUserConsent: parameters];
     
+    if (self.bidderTokenLoader == nil) {
+        self.bidderTokenLoader = [[YMABidderTokenLoader alloc] init];
+    }
```

Comment:
Does it make sense to make this static as opposed to allocating a new instance every instance of the adapter?

Reply:
It's OK to use it as static